src/BatteryStateOfChargeService.h

```diff
@@ -17,6 +17,15 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
 private:
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
-
+    double objectCurrent;
```

Comment:
This could use a better name, I don't really understand what it's being used for right now.